recipes-core/images/intel-aero-image.bb

```diff
@@ -38,6 +38,9 @@ IMAGE_INSTALL += "libavahi-client libavahi-glib"
 # Build tools
 IMAGE_INSTALL += "packagegroup-core-buildessential"
 
+# Development/Debug tools
+IMAGE_INSTALL += "valgrind strace gdb cgdb"
```

Comment:
I had a typo in my `find` command... we actually have all the dependencies for cgdb already. I still think it's not worth though.